server/users.go

```diff
@@ -104,6 +104,30 @@ func newUsersResponse(users []chronograf.User) *usersResponse {
 	}
 }
 
+// Chronograf User Roles
+const (
+	ViewerRoleName = "Viewer"
```

Comment:
all other string choices in chronograf are lower-case.  I'd say we should lowercase these as well for predictability.

Reply:
cool, done - thanks